metricbeat/module/kubernetes/state_namespace/_meta/data.json

```diff
@@ -0,0 +1,28 @@
+{
+    "@timestamp": "2019-03-01T08:05:34.853Z",
+    "event": {
+        "dataset": "kubernetes.state_namespace",
+        "duration": 115000,
+        "module": "kubernetes"
+    },
+    "kubernetes": {
+        "state_namespace": {
+            "created": {
+                "sec": 1691566337
+            },
+            "name": "kube-node-lease",
```

Comment:
I think reporting the name here would be redundant since it will be added anyways by the metadata Enricher. However maybe you can added under `kubernetes.namespace` directly and this would either be replaced by the Enricher or just remain there if the Enricher is disabled.

Reply:
Unfortunately, this is not true, because the `kubernetes.namespace.name` gets overwritten. I explain why in this comment: https://github.com/elastic/beats/pull/36406#discussion_r1304173303.